src/Data/Swagger/SchemaOptions.hs

```diff
@@ -41,3 +44,20 @@ defaultSchemaOptions = SchemaOptions
   , unwrapUnaryRecords = False
   }
 
+-- | Convert 'Aeson.Options' to 'SchemaOptions'.
+--
+-- Specifically the following fields get copied:
```

Comment:
It would be more interesting which options do NOT get copied, as to know what to look out for.

Reply:
Updated documentation.